tools/qtc_templates/kuesa/project/empty_quick/main.qml

```diff
@@ -0,0 +1,67 @@
+import Qt3D.Core %{Qt3DCoreVersion}
+import Qt3D.Render %{Qt3DVersion}
+import Qt3D.Input %{Qt3DInputVersion}
+import Qt3D.Extras %{Qt3DVersion}
+import QtQuick %{QtQuickVersion}
+import QtQuick.Scene3D %{Scene3DVersion}
+import Kuesa %{KuesaVersion} as Kuesa
+
+Item {
+    id: root
+
+    Scene3D {
+        id: scene3d
+        anchors.fill: parent
+        aspects: ["render", "input", "logic", "animation"]
+
+        Kuesa.SceneEntity {
```

Comment:
I'm still pondering about that one... wouldn't it be a better practice to put this entity in its own qml file? This way we'd split between QtQuick and Kuesa code. It's what we seem to do most of the time AFAIK.

Reply:
done